.github/workflows/test.yml

```diff
@@ -24,5 +24,7 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - name: Test Format
         run: cargo fmt -- --check
+      - name: Add wasm32 target
+        run: rustup target add wasm32-unknown-unknown
```

Comment:
If I am not mistaken, this should not be necessary? `actions-rs/toolchain@v1` already installs this target, no?